djstripe/models.py

```diff
@@ -442,27 +442,25 @@ class Event(StripeEvent):
 
     @property
     def message(self):
-        """ The event's data if the event is valid, None otherwise."""
-
+        """The event's data if the event is valid, None otherwise."""
         return self.webhook_message if self.valid else None
 
     def validate(self):
         """
+        Make an API call to Stripe to re-download the Event data.
+
         The original contents of the Event message comes from a POST to the webhook endpoint. This data
         must be confirmed by re-fetching it and comparing the fetched data with the original data. That's what
         this function does.
```

Comment:
Remove "That's what this function does." and "If valid, it then marks this record's valid flag to True or False."